### PR TITLE
Documentation Cleanup Round 1.

### PR DESCRIPTION
**Pull request by bhamodi**
This PR just does some basic cleanup of documentation. Went through and made things more consistent as well as more understandable.

### CODE REVIEW

File: docs/docs/10.1-animation.md [hunk 1]
```diff
@@ -86,7 +87,7 @@ You'll notice that animation durations need to be specified in both the CSS and
 
 `ReactCSSTransitionGroup` provides the optional prop `transitionAppear`, to add an extra transition phase at the initial mount of the component. There is generally no transition phase at the initial mount as the default value of `transitionAppear` is `false`. The following is an example which passes the prop `transitionAppear` with the value `true`.
 
-```javascript{3-5}
```
Comment (zpao) [inline docs/docs/10.1-animation.md]: Why did you change these? They're currently used for line highlighting.

Comment (bhamodi) [inline docs/docs/10.1-animation.md]: Hey @zpao - I changed those for two reasons. The first was that I couldn't find any documentation on markdown compiling only on certain line numbers. Secondly, it didn't make sense to me why we would want to only highlight certain lines in a code block? Please do correct me / point me in the right direction.

Comment (zpao) [inline docs/docs/10.1-animation.md]: I made it up :) http://zpao.com/posts/adding-line-highlights-to-markdown-code-fences/

As for why… it's usually to highlight a specific change or something important - that's the main way we use it in the tutorial. Most of the uses in here… not actually that valuable. The first on the page one is probably the most valuable since it's a larger code block.

Comment (bhamodi) [inline docs/docs/10.1-animation.md]: Wow that's impressive!

I'll go ahead and put back the ranges on the lines that should be emphasized.

**Comment by bhamodi** [comment]
Is this good now @zpao?

**Comment by zpao** [comment]
Looks great! Would you mind squashing the commits?

**Comment by bhamodi** [comment]
Squashed! :)

**Comment by zpao** [comment]
Thanks!
